.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode
+amun/__pycache__/
```

Comment:
this is thoth specific?

Reply:
yes

Reply:
so just `__pycache__` should work (all the `__pycache__` directories, which makes more sense anyway).